linux/sound.cpp

```diff
@@ -305,9 +305,10 @@ int CSound::process ( jack_nframes_t nframes, void* arg )
                 jack_midi_event_get ( &in_event, in_midi, j );
 
                 // copy packet and send it to the MIDI parser
-                // clang-format off
-// TODO do not call malloc in real-time callback
-                // clang-format on
+                // TODO: BEGIN
+                /* do not call malloc in real-time callback
+                 */
+                // TODO: END
```

Comment:
```suggestion
                // TODO: BEGIN
                /* do not call malloc in real-time callback */
                // TODO: END
```

Reply:
OK, so single comments also the closing */ on the same line ?
But that is not according your [proposal](https://github.com/jamulussoftware/jamulus/issues/2591#issuecomment-1100840078) ;=))

Reply:
I know. But after seeing it in real code, it seemed very strange.